lib/Internal/FileTask.php

```diff
@@ -65,13 +65,14 @@ public function run(Environment $environment) {
                     case "x+":
                     case "c":
                     case "c+":
+                    case "e":
```

Comment:
Why did you add `e` here?

Reply:
I think I've misinterpreted the code here (and expected that `e` was being passed down, I'll remove :)